src/GraphQL/TypeApi.hs

```diff
@@ -46,6 +47,27 @@ instance Exception QueryError
 queryError :: forall m a. MonadThrow m => Text -> m a
 queryError = throwM . QueryError
 
+-- | Object field separation operator.
+--
+-- Use this to provide handlers for fields of an object.
+--
+-- Say you had an object with \"foo\" and \"bar\" fields, e.g.
```

Comment:
maybe say "e.g the following GraphQL type"? - I don't think many people actually know what graphql types look like by heart